contracts/pcv/uniswap/UniswapPCVDeposit.sol

```diff
@@ -155,17 +155,26 @@ contract UniswapPCVDeposit is IUniswapPCVDeposit, PCVDeposit, UniRef {
         5. rETH = sqrt(k / P)
         
         and rFEI = k / rETH by 1.
+
+        Finally scale the resistant reserves by the ratio owned by the contract
      */
     function resistantBalanceAndFei() public view override returns(uint256, uint256) {
-        (uint256 feiInPool, uint256 ethInPool) = getReserves();
-        Decimal.D256 memory ethFei = readOracle();
+        (uint256 feiInPool, uint256 otherInPool) = getReserves();
```

Comment:
Is the order of the addresses retrieved here always going to have fei first?

Reply:
Yes